site_scons/site_tools/doneapi.py

```diff
@@ -0,0 +1,88 @@
+"""SCons.Tool.doneapi
+
+Hack to support oneapi version of Intel compilers
+
+"""
+import sys
+import os
+
+
+is_linux = sys.platform.startswith('linux')
+
+# pylint: disable=no-name-in-module
+# pylint: disable=import-error
+if is_linux:
+    import SCons.Tool.gcc
+import SCons.Util
+import SCons.Warnings
+import SCons.Errors
+# pylint: enable=import-error
+# pylint: enable=no-name-in-module
+# pylint: disable=no-member
+# pylint: disable=too-few-public-methods
+
+if not is_linux:
+    raise SCons.Errors.UserError("This tool is only supported on Linux")
```

Comment:
This is true for our whole build isn't it?

Reply:
True